client/flutter/lib/pages/question_index.dart

```diff
@@ -62,44 +66,79 @@ class _QuestionIndexPageState extends State<QuestionIndexPage> {
               )
             : SliverToBoxAdapter(
                 child: Padding(
-                  padding: const EdgeInsets.all(48.0),
-                  child: CupertinoActivityIndicator(),
-                )
-
-              )
+                padding: const EdgeInsets.all(48.0),
+                child: CupertinoActivityIndicator(),
+              ))
       ],
       title: widget.title,
     );
   }
 
-  Widget _buildQuestion(QuestionItem questionItem) {
-    return Padding(
-      padding: const EdgeInsets.all(8.0),
+  // flutter_html supports a subset of html: https://pub.dev/packages/flutter_html
+}
+
+class QuestionTile extends StatefulWidget {
+  const QuestionTile({
+    @required this.questionItem,
+  });
+
+  final QuestionItem questionItem;
+
+  @override
+  _QuestionTileState createState() => _QuestionTileState();
+}
+
+class _QuestionTileState extends State<QuestionTile>
+    with TickerProviderStateMixin {
+  bool isExpanded;
+  AnimationController rotationController;
+
+  @override
+  void initState() {
+    super.initState();
+    isExpanded = false;
+    rotationController = AnimationController(
+        duration: const Duration(milliseconds: 200), vsync: this, lowerBound: 0, upperBound: 45);
+
+    
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: Colors.white,
       child: Column(children: <Widget>[
-        Divider(),
+        Divider(
+          height: 1,
+        ),
         ExpansionTile(
-          key: PageStorageKey<String>(questionItem.title),
-          trailing: Icon(
-            Icons.add_circle_outline,
-            color: Colors.black,
-          ),
-          title: Padding(
-            padding: const EdgeInsets.all(8.0),
-            child: html(questionItem.title),
+          onExpansionChanged: (v) {
+            if(v){
+              rotationController.forward();
+            }else{
+              rotationController.reverse();
+            }
+          },
+          key: PageStorageKey<String>(widget.questionItem.title),
+          trailing: AnimatedBuilder(
+            animation: rotationController,
+            builder: (c,w){
+              return Transform.rotate(angle: rotationController.value, child: Icon(Icons.add_circle_outline, color: Colors.grey),);
```

Comment:
Also, `child` of `AnimatedBuilder` should be used for the `Icon` instead of inlining below (stated as good practise here: https://api.flutter.dev/flutter/widgets/AnimatedBuilder-class.html )